ansible_3par_docker_plugin/README.md

```diff
@@ -197,11 +202,24 @@ Once complete you will be ready to start using the HPE 3PAR Docker Volume Plug-i
      $ cd python-hpedockerplugin/ansible_3par_docker_plugin
      $ ansible-playbook -i hosts uninstall/uninstall_hpe_3par_volume_driver_etcd.yml
      ```
-
+    * Uninstall plugin on standalone environment:
+     ```
+     $ cd ~
+     $ cd python-hpedockerplugin/ansible_3par_docker_plugin
+     $ ansible-playbook -i hosts_standalone_nodes uninstall/uninstall_standalone_hpe_3par_volume_driver.yml
+     ```
+     
+     * Uninstall plugin along with etcd standalone environment:
+     ```
+     $ cd ~
+     $ cd python-hpedockerplugin/ansible_3par_docker_plugin
+     $ ansible-playbook -i hosts_standalone_nodes uninstall/uninstall_standalone_hpe_3par_volume_driver_etcd.yml
+     ```
```

Comment:
@bhagyashree-sarawate Do add a pointer for Install plugin on standalone environment